WordPress/WordPressTest/KeychainTest.m

```diff
@@ -1,6 +1,7 @@
+// TODO: Port to yet to create SFHFKeychainUtils test package
```

Comment:
Waiting for feedback on this PR. Then, if the setup remains the same, I'll open an issue to remember to do this.

Reply:
https://github.com/wordpress-mobile/WordPress-iOS/issues/24201